### PR TITLE
[python] expose VideoInfoTag and MusicInfoTag

### PR DESCRIPTION
**Pull request by tamland**
This is pretty useless now, but will be needed for #6316/#1654

### CODE REVIEW

**Comment by mkortstiege** [comment]
Just out of curiosity, do you plan to interact with `setInfo()` at some point or is this pure get?

**Comment by tamland** [comment]
Don't know... Only get I guess. Context item addons will be passed a copy of the listitem so `setInfo` won't do anything.

File: xbmc/interfaces/swig/AddonModuleXbmcgui.i [hunk 1]
```diff
@@ -43,6 +43,8 @@ using namespace xbmcgui;
 // not part of what swig parses.
 %feature("knownbasetypes") XBMCAddon::xbmcgui "AddonClass,AddonCallback"
 
+%feature("knownapitypes") XBMCAddon::xbmcgui "XBMCAddon::xbmc::InfoTagVideo,xbmc::InfoTagMusic"
```
Comment (Montellese) [inline xbmc/interfaces/swig/AddonModuleXbmcgui.i]: Did you leave the `XBMCAddo::` namespace for `xbmc::InfoTagMusic` out on purpose?

Comment (tamland) [inline xbmc/interfaces/swig/AddonModuleXbmcgui.i]: I wrote this in October so I have no idea. Typo probably, but it still worked for some reason

Comment (tamland) [inline xbmc/interfaces/swig/AddonModuleXbmcgui.i]: I remember now. There's weird bug in the generator I believe, that I could never resolve. It refuse to recognize the type if the XBMCAddon namespace is in the second parameter. Possibly it inherits the top namespace while parsing it or something?

**Pull request closed without merging**